.github/workflows/feature_branch_deletion.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature branch deletion cleanup
+on:
+  delete:
+    branches:
+      - feature_**
```

Comment:
Elsewhere, you named your branch `feature-project-integrity`, but this syntax implies a canonical naming of feature branches with `feature_*`, which creates a conflict of dashes (-) versus underscores (_), and you should declare a winner.

Reply:
>.< lol...